### ISSUE TITLE
Remove rounded corners from Pick a Date/Time row

### ISSUE THREAD

**Issue by sevaan** [bwinton/SnoozeTabs#130]
The Pick a Date/Time row has rounded corners at the bottom which should be removed.

<img width="702" alt="screen shot 2017-02-01 at 11 39 22 am" src="https://cloud.githubusercontent.com/assets/473531/22516217/1b18340e-e873-11e6-83ea-dca8bd4430d6.png">

**Comment by lmorchard** [comment]
Is this actually an issue with PR #120? I'm not seeing this on master branch

**Comment by sevaan** [comment]
Yes it is. Sorry for not specifying. Blake set me up with a build of it, so the issues I've submitted in the last ten minutes are using that build.

**Comment by lmorchard** [comment]
Might be better to just comment over there, since that PR's not merged yet
